test/README-OPENSHIFT4.md

```diff
@@ -0,0 +1,52 @@
+# Setup for OpenShift 4 tests #
+
+## Cluster and Operator setup ##
+
+### Set up a local OpenShift 4 cluster using CodeReady Containers ###
+
+Install the OpenShift CLI tool:
+
+```shell
+brew install openshift-cli
+```
+
+Download CodeReady Containers: [https://cloud.redhat.com/openshift/install/crc/installer-provisioned](https://cloud.redhat.com/openshift/install/crc/installer-provisioned). Use the credentials `Red Hat / Openshift` to authenticate to Red Hat.
```

Comment:
Should we reference a 1Password account to use?

Reply:
Good point, will add 👍